redisai/pipelinedClient.go

```diff
@@ -0,0 +1,167 @@
+package redisai
```

Comment:
I'm not sure that I agree with the approach of a separate interface pipelined clients - I would have rather seen something like

```
c := client.Connect(...)
p := c.pipeline()
p.GetTensor(...)
...
d, err := p.execute()
```

Reply:
Also, I'm missing an example on how to use this.

Reply:
@itamarhaber can you check the https://github.com/filipecosta90/redisai-go/tree/itamarhaber/examples folder. 
what do you think?